peekingduck/pipeline/nodes/preprocess/brightness.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2021 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Adjusts the brightness of an image.
+"""
+
+
+from typing import Any, Dict
+
+import cv2
+import numpy as np
+
+from peekingduck.pipeline.nodes.node import AbstractNode
+
+
+class Node(AbstractNode):
+    """Adjusts the brightness of an image, by adding a bias/
+    `beta parameter <https://docs.opencv.org/4.x/d3/dc1/tutorial_basic_
+    linear_transform.html>`_.
+
+    Inputs:
+        |img|
+
+    Outputs:
+        |img|
+
+    Configs:
+        beta (:obj:`int`): **[-100,100], default = 0**. |br|
```

Comment:
1) I used 100 because I saw this in a number of opencv related brightness adjustment articles. [This](https://docs.opencv.org/3.4/d3/dc1/tutorial_basic_linear_transform.html) is one example.
2) There is not much point to use a beta of 255 because all the pixels will turn out white (255), or vice versa.

Reply:
Sure, perhaps this range can be adjusted later based on user feedback if any.

Because right now, we are limiting it to a range of "arbitrary recommended" values instead of valid values. +110 won't make all pixels white and -110 won't make all pixels black but is prohibited by the current code.